docs/api/migration-guides/qiskit-runtime-examples.mdx

```diff
@@ -534,7 +534,7 @@ options that do not have a legacy alternative to migrate from, but can
 help improve your performance and results. For more information, refer
 to the following:
 
--   [Error mitigation tutorial](https://learning.quantum-computing.ibm.com/tutorial/error-suppression-and-error-mitigation-with-qiskit-runtime)
+-   [Error mitigation tutorial](../tutorial/error-suppression-and-error-mitigation-with-qiskit-runtime)

```

Comment:
This is a bad relative link. Relative links only work if the site is `docs.quantum-computing`, not `learning.quantum-computing`.

Reply:
There are a couple more instances of this in the PR.

Reply:
Oh, yes, sorry, had to clarify that, I was scratching my head because of that difference, good to know that, Thank You